src/test/java/org/assertj/core/internal/bigdecimals/BigDecimals_assertHasScale_Test.java

```diff
@@ -0,0 +1,50 @@
+package org.assertj.core.internal.bigdecimals;
+
+import org.assertj.core.internal.BigDecimalsBaseTest;
+import org.assertj.core.util.AssertionsUtil;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+
+import static java.lang.String.format;
+import static java.math.BigDecimal.ONE;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+
+class BigDecimals_assertHasScale_Test extends BigDecimalsBaseTest {
```

Comment:
follow the GIVEN WHEN THEN pattern as in `Iterables_assertContainsOnly_Test`

Reply:
I was kind of confused as to what to do because of some tests that I looked at. If what I did was incorrect just let me know :)